201-redis-premium-vnet/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
remove this file since we have an existing vnet that you can use

Reply:
Should I reference an existing vnet template anywhere, such as in the README?